extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -290,6 +290,20 @@ export class SubscriptionService {
       return this.timer_
         .timeoutPromise(timeout, subscriptionPlatform.getEntitlements())
         .then(entitlement => {
+          if (
+            entitlement.granted &&
+            this.cryptoHandler_.isDocumentEncrypted() &&
+            !entitlement.decryptedDocumentKey
+          ) {
+            const x =
+              subscriptionPlatform.getServiceId() == 'local' ? user() : dev();
+            x.error(
+              TAG,
+              'Subscription granted and encryption enabled, ' +
```

Comment:
Let's add `serviceId` in the message. E.g. `${serviceId}: Entitlements granted, but ...`

Reply:
Done.